hoot-core/src/main/cpp/hoot/core/algorithms/WayJoiner2.cpp

```diff
@@ -165,110 +127,6 @@ void WayJoiner2::_joinParentChild()
   }
 }
 
-void WayJoiner2::_joinSiblings()
-{
-  LOG_TRACE("Joining siblings...");
-
-  WayMap ways = _map->getWays();
-  // Get a list of ways that still have a parent
-  map<long, deque<long>> w;
-  //  Find all ways that have a split parent id
-  for (WayMap::const_iterator it = ways.begin(); it != ways.end(); ++it)
-  {
-    WayPtr way = it->second;
-    if (way->hasPid())
-    {
-      long parent_id = way->getPid();
-      w[parent_id].push_back(way->getId());
-    }
-  }
-  //  Rejoin any sibling ways where the parent id no longer exists
-  for (map<long, deque<long>>::iterator map_it = w.begin(); map_it != w.end(); ++map_it)
-  {
-    deque<long>& way_ids = map_it->second;
-    LOG_VART(way_ids);
-    while (way_ids.size() > 1)
-      _rejoinSiblings(way_ids);
-  }
-}
-
-void WayJoiner2::_joinUnsplitWaysAtNode()
```

Comment:
This function is just moved to be in the same order it is found within the class declaration.